src/api/main.py

```diff
@@ -0,0 +1,65 @@
+import os
+from os.path import join
+import errno
+import shutil
+from subprocess import call
+import json
+import uuid
+
+from flask import Flask, jsonify
+
+from rwd.Rapid_Watershed_Delineation import Point_Watershed_Function
+
+app = Flask(__name__)
+
+
+@app.route('/rwd/<lat>/<lon>', methods=['GET'])
+def run_rwd(lat, lon):
+    """
+    Runs RWD for lat/lon and returns the GeoJSON
+    for the computed watershed boundary.
+    """
+    # TODO This function needs to be made more robust to failures.
+
+    data_path = '/opt/rwd-data'
+
+    # Create a temporary directory to hold the output
+    # of RWD
+    output_path = join('/tmp/rwd/', str(uuid.uuid4()))
+    if os.path.exists(output_path):
+        shutil.rmtree(output_path)
+    os.makedirs(output_path)
+
+    Point_Watershed_Function(
+        lat,
+        lon,
+        '1',
```

Comment:
This was designed so that this could be a user input.  We had in mind some sort of interface dialog accompanying a user clicking on the map being able to adjust whether snapping should be active, and if it was active what the maximum_snap_distance should be (in meters).  (If user units are different than meters a unit conversion may be needed).